### ISSUE TITLE
Problem importing splatmap

### ISSUE THREAD

**Issue by dr-ubin** [Zylann/godot_heightmap_plugin#145]
Hi,
This is a great plugin, but importing a splat map isn't working for me.  I am able to import the heightmap, but the splat map has no effect in the shader.  If I use a custom shader and output the splat texture from the shader, it is all red.  I am using a png file with the same dimensions as the heightmap.  If I manually copy the splat file into the terrain directory, I can see the change, but the texture placement has large geometric artifacts (I assume it's using an internal format after importing).  Any help is appreciated.  
Thanks

Steps to reproduce: 
- Start a new empty project
- Enable the heightmap plugin
- Set the terrain directory
- Click "Terrain > Import maps" and select files and click Import

Specs
Mac OS X 10.14.4
Radeon Pro 560
Heightmap - current master branch
Godot 3.2

**Comment by Zylann** [comment]
> If I manually copy the splat file into the terrain directory, I can see the change, but the texture placement has large geometric artifacts (I assume it's using an internal format after importing).

Are those geometric artifact from the 4th channel? (i.e the alpha channel)

**Comment by Zylann** [comment]
The importer should be fixed in 9b27a733b464c0ac25cd3d07751e07bef94daa46

**Comment by dr-ubin** [comment]
Thanks - the importer works now.  I am still seeing artifacts but I'll try to get to the bottom of it tomorrow.  I am using Gimp on a Mac and I couldn't find anything about alpha compression but I did try setting the compression level to zero, but the result was the same.  I also tried adding an alpha channel of all 0s or 1s but saw more or less the same effects.

**Comment by Zylann** [comment]
![image](https://user-images.githubusercontent.com/1311555/79133214-90297a00-7da3-11ea-917e-02db59037e4a.png)

`Save color values from transparent pixels` is probably what I meant. I can't click on it though, dunno why.

**Comment by dr-ubin** [comment]
Thanks, I am making some progress.  I think some of my tests were thrown off by issues with the textures I was using, so I went back to the demo project to experiment with those textures.  Also, I believe the artifacts had to do with varying the splatmap in the lite shader so I am using a modified version for now.  

One thing I am confused about is when I painted the following pattern in the heightmap plugin 
![Screen Shot 2020-04-13 at 7 24 10 PM](https://user-images.githubusercontent.com/3057470/79178498-7738bd00-7dd3-11ea-937d-b2f15bf6f1aa.png)
And then I look at the resulting splatmap file in Gimp, it looked like this:
![Screen Shot 2020-04-13 at 7 24 53 PM](https://user-images.githubusercontent.com/3057470/79178576-9a636c80-7dd3-11ea-8ad3-5f19aec9e997.png)

Going the other way is working now- I can modify or import the splatmap and see the results.
